packages/docs-page/server/loaders/remote-content.ts

```diff
@@ -95,22 +104,28 @@ export default class RemoteContentLoader implements DataLoader {
     if (!this.opts.mainBranch) this.opts.mainBranch = 'main'
     if (!this.opts.scope) this.opts.scope = {}
     if (!this.opts.remarkPlugins) this.opts.remarkPlugins = []
+
+    this.navDataPrefix = this.opts.navDataPrefix || this.opts.basePath
```

Comment:
suggestion: convention is for everything to be read from `this.opts`, so we can use a similar default approach above and assign to `this.opts.basePath` if it is unset. No need for the extra class property

Reply:
Good call! I'll do that

Reply:
https://github.com/hashicorp/react-components/pull/583/commits/445d4fbef12bddd77ee2af5997b8f94518e765e0